extensions/parquet/compression/src/main/java/io/deephaven/parquet/compress/DeephavenCompressorAdapterFactory.java

```diff
@@ -151,17 +149,9 @@ public BytesInput decompress(final InputStream inputStream, final int compressed
             } else {
                 decompressor = null;
             }
-            try {
-                // Note that we don't close the decompressed stream because doing so may return the decompressor to the
-                // pool
-                final InputStream buffered = ByteStreams.limit(IOUtils.buffer(inputStream), compressedSize);
-                final CompressionInputStream decompressed = compressionCodec.createInputStream(buffered, decompressor);
-                return BytesInput.copy(BytesInput.from(decompressed, uncompressedSize));
-            } finally {
-                if (decompressor != null) {
-                    decompressor.reset();
```

Comment:
Are we failing to reset the decompressor now?

Reply:
I removed this step because this was not required.
Basically, if we are picking up the decompressor from the decompressorHolder, we reset it in line 140.
And if we are getting it from the CodecPool, the decompressor is already reset at the time of adding to the pool.
I can add a comment explaining this.